### PR TITLE
small bug fix

### PR DESCRIPTION
**Pull request by meguiraun**
remove unused method call, missed from previous pr

### CODE REVIEW

File: mxcube3/routes/Login.py [hunk 1]
```diff
@@ -125,7 +125,6 @@ def loginInfo():
 
     login_info = login_info["loginRes"] if login_info is not None else {}
     login_info = limsutils.convert_to_dict(login_info)
-    limsutils.update_mxcube_session(login_id, login_info)
```
Comment (marcus-oscarsson) [inline mxcube3/routes/Login.py]: Hm, how is this unused ?

Comment (meguiraun) [inline mxcube3/routes/Login.py]: because it is done in the select_proposal

Comment (marcus-oscarsson) [inline mxcube3/routes/Login.py]: It looks to me like some logic that once were in limsutils is now missing. The method update_mxcube_session made sure that the mxcube Session object was kept in sync when for instance the server is restarted.

Did you put it somewhere else ?, the data paths will not be correct if this information is not kept.

Comment (meguiraun) [inline mxcube3/routes/Login.py]: see https://github.com/mxcube/mxcube3/blob/master/mxcube3/routes/limsutils.py#L80

Of course... there could be a bug

Comment (marcus-oscarsson) [inline mxcube3/routes/Login.py]: Hm, well okey and what calls select_proposal ?, what hapends for instance when the server is restarted and we reconnect the client ? How is the mxcube Session object updated ?

Well in anyways, I guess Ill accept this and we have to fix it later ...

Comment (meguiraun) [inline mxcube3/routes/Login.py]: select_proposal is called in loginInfo, I think this is the same solution as the other method, just taking into account user/proposal login. Did you try running?

No need to accept per se if you do not agree. I can modify on this pr.

Comment (marcus-oscarsson) [inline mxcube3/routes/Login.py]: Hm, okey Ill merge now and I will try on the beamline later.

Thanks Mikel

Comment (meguiraun) [inline mxcube3/routes/Login.py]: thanks, let me know if you have issues, this is a tricky part and different in our sites.

**Pull request merged**